src/update-manager.ts

```diff
@@ -91,14 +91,52 @@ export default class UpdateManager {
     return( uri.slice(0,4) === 'http' )
   }
 
+/** Remove from the store HTTP authorization metadata
+* The editble function below relies on copies we have in the store
+* of the results of previous HTTP transactions. Howver, when
+* the user logs in, then that data misrepresents what would happen
+* if the user tried again.
+*/
+async flagAuthorizationMetadata () {
```

Comment:
Does this have to be called manually to trigger an invalidation where needed or is it called somewhere from rdflib itself?

Reply:
It has to be called by the app when the user logs in.  Or, I guess from the login code.  But rdflib itself does not have a spot in the login flow.